<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Rushis-HP" failures="1" tests="10" name="testCases.SignInPagetest" time="27.110" errors="0" timestamp="2024-10-01T16:38:22 CDT" skipped="0">
  <testcase classname="testCases.SignInPagetest" name="checkValidAndInvalidLoginTest" time="2.616"/>
  <testcase classname="testCases.SignInPagetest" name="checkValidAndInvalidLoginTest" time="3.209"/>
  <testcase classname="testCases.SignInPagetest" name="checkValidAndInvalidLoginTest" time="2.762"/>
  <testcase classname="testCases.SignInPagetest" name="checkValidAndInvalidLoginTest" time="2.902"/>
  <testcase classname="testCases.SignInPagetest" name="checkValidAndInvalidLoginTest" time="2.684"/>
  <testcase classname="testCases.SignInPagetest" name="testLoginWithoutUsername" time="2.407"/>
  <testcase classname="testCases.SignInPagetest" name="testLoginWithoutUsername" time="2.539"/>
  <testcase classname="testCases.SignInPagetest" name="testLoginWithoutUsername" time="2.439"/>
  <testcase classname="testCases.SignInPagetest" name="checkUsernameDisplayAfterLogin" time="2.996">
    <failure message="expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertTrue(Assert.java:57)
at org.testng.Assert.assertTrue(Assert.java:67)
at testCases.SignInPagetest.checkUsernameDisplayAfterLogin(SignInPagetest.java:96)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- checkUsernameDisplayAfterLogin -->
  <testcase classname="testCases.SignInPagetest" name="checkMovingToRegisterPageFromLoginPage" time="2.556"/>
</testsuite> <!-- testCases.SignInPagetest -->
